dataactbroker/handlers/accountHandler.py

```diff
@@ -387,6 +387,10 @@ def updateUser(self, system_email):
         if requestDict.exists("is_active"):
             is_active = bool(requestDict.getValue("is_active"))
             if not self.isUserActive(user) and is_active:
+                # Reset password count to 0
+                self.resetPasswordCount(user)
+                # Reset last login date so the account isn't expired
+                self.interfaces.userDb.updateLastLogin(user, unlock_user=True)
```

Comment:
I put the unlock_user param for that reason. That flags it to be set to None so it's empty in the database. That way it doesn't look like they've logged in recently. With a blank last_login_date the only reasons are that they've never logged in before or they've just been unlocked.